traffic_ops/app/lib/API/User.pm

```diff
@@ -277,6 +285,7 @@ sub current {
 			@data, {
 				"id"              => "0",
 				"username"        => $current_username,
+				"tenantId"	  => $self->current_user_tenant(),
```

Comment:
at some point I think ldap will be completely removed as we add the token authentication microservice but for now, i think if you login as ldap, they get the read-only role and the root tenant...